test/integration/connect/envoy/helpers.bash

```diff
@@ -351,6 +351,27 @@ function assert_service_has_healthy_instances {
   [ "$status" -eq 0 ]
 }
 
+function check_intention {
```

Comment:
Nice! The collection of fns based off this seem pretty good to have on hand 👍

Reply:
I was surprised we didn't already do anything with intentions in these integration tests. Now we can.